frontend/elements/src/entry/home/TOSORT/contet-wimg.ts

```diff
@@ -10,45 +9,59 @@ export class _ extends LitElement {
         width: 1676px;
         height: 540px;
         background-color:#ffffff ;
-        margin-left:122px;
+        margin-left:80px;
         border-radius: 25px;
         box-shadow: 0 3px 20px 0 rgba(0, 0, 0, 0.16);
  
- 
      }
 
-     ::slotted([slot=title]){
-        font-size: 64px;
-        font-weight: 900;
-        color:#5662a3;
-        text-align: center;
-       }
     
-      img-ui{
-        width: 960px;
-        height: 540px;
+     .img{
+          width: 940px;
+          height: 540px;
+          display:block;
+          border-radius: 25px 0px 0px 25px;
+          // background-image: url("Sara-Halberstadt.jpg");
```

Comment:
this shouldn't be in the codebase at all.. not going to worry about erasing it from git history but please delete

Reply:
Deleted and fixed